source/routing/specifying-a-routes-model.md

```diff
@@ -204,7 +202,31 @@ will always execute the model hook.
 If your data represented by your model is being updated frequently, you may
 want to refresh it periodically:
 
-<a class="jsbin-embed" href="http://jsbin.com/sefuv/2/embed?js">JS Bin</a><script src="http://static.jsbin.com/js/embed.js"></script>
+```app/routes/pull-requests.js
+export default Ember.Route.extend({
+  model() {
+    var url = 'https://api.github.com/repos/emberjs/ember.js/pulls';
+    return Ember.$.getJSON(url).then(function(data) {
```

Comment:
`Ember.$.getJSON` is used in other examples on this page – I agree with @michaelrkn that it warrants a new PR.

Sorry for late response, quite busy this week.